python/GafferArnoldTest/IECoreArnoldPreviewTest/RendererTest.py

```diff
@@ -1427,15 +1427,14 @@ def testStepSizeAttribute( self ) :
 			"sphere" : IECore.SpherePrimitive(),
 			"mesh" : IECore.MeshPrimitive.createPlane( IECore.Box2f( IECore.V2f( -1 ), IECore.V2f( 1 ) ) ),
 			"curves" : IECore.CurvesPrimitive.createBox( IECore.Box3f( IECore.V3f( -1 ), IECore.V3f( 1 ) ) ),
-			"volumeProcedural" : IECore.ExternalProcedural(
-				"volume_vdb.so",
+			"procedural" : IECore.ExternalProcedural(
+				"volume",
 				IECore.Box3f( IECore.V3f( -1 ), IECore.V3f( 1 ) ),
-				IECore.CompoundData( {
-					"ai:nodeType" : "volume",
-				} )
 			),
 			"regularProcedural" : IECore.ExternalProcedural(
-				"test.so",
+				# There aren't any procedurals we rely on being installed, but we can create any regular
+				# node the same way we create a procedural
+				"cone", 
```

Comment:
Can we just use "procedural" here?

On a related note, I see that Arnold 5 supports setting "step_size" on polymesh now, so the "convert meshes with a step size to boxes" thing that we're testing here could be removed now. Would be nice to add that before merging.

Reply:
> Can we just use "procedural" here?

That seems misleading?  If someone actually writes an external procedural, it won't create a node type named "procedural", it will create a node type named "MyCoolThing" or whatever.  The only reason to use "procedural" in an ExternalProcedural is for ass archives, not procedurals.  I guess I could test the ass archive case, but then I should also change the name from "regularProcedural" to "assArchive".

Reply:
> That seems misleading?

I don't think it's as misleading as creating a cone!

> The only reason to use "procedural" in an ExternalProcedural is for ass archives, not procedurals. I guess I could test the ass archive case

Sounds good.

> but then I should also change the name from "regularProcedural" to "assArchive".

OK.

Reply:
In the end, I double checked, and we can get rid of the mesh->box step size thing completely, so I just removed the troublesome line about testing a default procedural, since I guess we're no longer concerned about screwing with procedurals by turning them into meshes.